clarkio/ttv-chat-light
# Post bot responses

## Purpose

- This taps into the bot conversation by listening for a response after sending a command to the bot. It will wait x amount of seconds before checking for new messages in the bot conversation. Once it does it will read the last message and share it in the Twitch chat.

## Does this introduce a breaking change?




## Pull Request Type

What kind of change does this Pull Request introduce?




## How to Test

1. run the app
2. enter a command in chat
3. verify a message for the bot response is shared in twitch chat
